### PR TITLE
remove Astro and unneeded dependencies

### PR DESCRIPTION
**Pull request by romainmenke**
This is very barebones but fully embraces native node:
- `http` for dev server
- `child_process` to render updates without restarting the process
- tagged template literals instead of a framework like Astro

New dependencies :
- Added `glob` to list the public files that need to be copied over.
- Added `postcss-cli` to easily and correctly build the CSS.

### CODE REVIEW

**Comment by romainmenke** [comment]
fixes : https://github.com/csstools/cssdb/issues/178

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -19,4 +19,4 @@ jobs:
           node-version: ${{ matrix.node }}
 
       - run: npm ci --ignore-scripts
-      - run: npm run test --if-present
+      - run: npm run test
```
Comment (Antonio-Laguna) [inline .github/workflows/test.yml:22]: Is this intended?

Comment (romainmenke) [reply .github/workflows/test.yml:22]: Yes, this repo doesn't use workspaces (afaik) and with `--if-present` we could have false positives after accidentally removing or renaming `"test"` in `package.json`.

File: src/components/feature.mjs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import { renderCaniuseFeature } from './caniuse-feature.mjs';
+import { renderFeaturePolyfills } from './feature-polyfills.mjs';
+
+const stages = [
+	'stage-0-aspirational',
+	'stage-1-experimental',
+	'stage-2-allowable',
+	'stage-3-embraced',
+	'stage-4-standardized',
+]
```
Comment (Antonio-Laguna) [inline src/components/feature.mjs:12]: Super minor (might be elsewhere) but this is lacking semi colon which is not a big deal but just flagging for consistency, perhaps it should be enforced?

File: src/components/feature.mjs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+				</p>
+			</header>
+			<p class="cssdb-feature-docs">
+				<a
+					href="${feature.specification}"
+					aria-label="Specification for ${escapeHTML(cleanTitle)}"
+				>
+					<img
+						src="https://img.shields.io/badge/Spec-005A9C?logo=w3c&style=flat-square"
+						alt="W3C Specification"
```
Comment (Antonio-Laguna) [inline src/components/feature.mjs:43]: Not sure what it was before but given the `aria-label` above I think this just adds noise? Should it just be empty so it's treated as decorative? 

Don't have a super-strong opinion here

File: src/components/footer.mjs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { html } from "../util/html.mjs";
+
+export function renderFooter() {
+	return html`
+		<footer>
+			<p>Want to contribute? Checkout the <a href="https://github.com/csstools/cssdb">cssdb on GitHub</a>.</p>
+			<p>Logos (except for Internet Explorer, BlackBerry, Baidu and QQ) are from <a href="https://github.com/alrra/browser-logos">alrra/browser-logos</a>. Check them out!</p>
```
Comment (Antonio-Laguna) [inline src/components/footer.mjs:7]: Should we update the disclaimer to include the oculus one?

**Review by Antonio-Laguna — APPROVED** [review]
This is looking great! Just left some minor questions but this is approved! Thanks for leading this!

File: src/components/feature.mjs (hunk 2 above)
Comment (romainmenke) [reply src/components/feature.mjs]: Good catch!

Unsure if we even have a linter setup here 🤔

File: src/components/feature.mjs (hunk 3 above)
Comment (romainmenke) [reply src/components/feature.mjs]: I agree.
All these labels were to verbose and contained too much repetition.
This definitely was annoying for screenreader users.

I've tried to clean this up a bit.

File: src/components/footer.mjs (hunk 4 above)
Comment (romainmenke) [reply src/components/footer.mjs]: Yes! added this :)

**Pull request merged**